defos/src/defos_mac.mm

```diff
@@ -110,6 +110,16 @@ void defos_set_window_size(float x, float y, float w, float h) {
     [window setFrame:NSMakeRect(x, win_y, w , h) display:YES];
 }
 
+void defos_set_window_icon(const char *icon_path)
+{
+    NSString *bundlePath = [[NSBundle mainBundle] resourcePath];
+    NSString *secondParentPath = [[bundlePath stringByDeletingLastPathComponent] stringByDeletingLastPathComponent];
```

Comment:
You could have gotten to this in a more easier way:
```objective-c
[[NSBundle mainBundle] bundlePath]
```

Reply:
ok, thx.